DE/Game.po

```diff
@@ -1719,7 +1719,7 @@ msgstr "Modell 02"
 #: /Game/Blueprint/ACF/Item/Weapon/ACFGunBase_SMG.Default__ACFGunBase_SMG_C.ItemInfo.Name
 msgctxt ",590B39C941F02271DB34EF8E862671DA"
 msgid "MP7A1冲锋枪"
-msgstr "MP7 SMG"
+msgstr "MP7A1 Maschinenpistole"
```

Comment:
Please dont change the Weapon Name. Weapon Names are fix from the Dev. See pinned post in Discord.